### DISCUSSION TITLE
Make module for Tizenbrew for samsung tizen tv's.

### DISCUSSION THREAD

**Discussion by tenda96** [DonutWare/Fladder#150] [Ideas]
### Summary

First, thank you for your project. I just love this interface. <3
However i was thinking if is possible for you to make a module for this:  [Tizenbrew](https://github.com/reisxd/TizenBrew/blob/main/docs/MODULES.md)
Because i've seen that one other dev have done jellyfin app for this module: at this [github](https://github.com/GlenLowland/jellyfin-web-npm-publish) and i want to use this interface on my samsung tv, and maybe some other people.
Let me know, thank you anyway! 

### Basic Example

TizenBrew is a way to experience modded websites (like [TizenTube](https://github.com/reisxd/TizenTube)) and you can install newer apps without having to deal with Tizen Studio multiple times on samsung tv's.

### Drawbacks

i don't think that will be some drawbacks, because is another way to use this interface.

**Comment by bazzadazza72** [comment]
I think that's a very good idea. I have two Samsung TVs myself, and I'd love a better interface than what the current Jellyfin app for Tizen provides.

What are your thoughts @PartyDonut? Should we add this to the roadmap?

**Comment by PartyDonut** [comment]
So Samsung TVs would then run the web version correct? I feel like that would probably not be the best experience how is video support when using a web version of the app?

Would something like this https://github.com/flutter-tizen/flutter-tizen#installation not be a better idea?

That being said if anyone is willing to make this work I have no objections, personally I do not own a Samsung TV so would be little help in that department.

**Comment by bazzadazza72** [comment]
> So Samsung TVs would then run the web version correct?

With what @tenda96 is suggesting, then yes. I've had a quick look at the documentation and it seems like it just serves the HTML directly to the TV.

Running a native version would be better IMO, though it'll take quite a lot of effort to get working reliably.

**Comment by PartyDonut** [comment]
Also take note that while the interface would kind of work (tested on AndroidTV). It is in no way optimized or even useable for d-pad control or TV layouts.

**Comment by tenda96** [comment]
From what i know, yes this is just a way to display some type of webpages... I was suggesting tizenbrew because of the easy method of install, you just use a browser from a pc and the user can install tizenbrew on the tv and after that, you can install all the module you need without any knowledge abour docker or other things... It's easier for the user :)

**Comment by bazzadazza72** [comment]
> I was suggesting tizenbrew because of the easy method of install, you just use a browser from a pc and the user can install tizenbrew on the tv and after that, you can install all the module you need without any knowledge abour docker or other things... It's easier for the user :)

That's all very true, but the app's layout isn't built for use on a TV, it's primarily built for use on desktop and mobile. It will likely require a lot of rework to both look good and function as it should.

**Comment by tenda96** [comment]
Okok no problem, understood. I will just use the jellyfin app on Samsung and keep using fladder on other devices ☺️
However, thanks for the quick reply, have a nice day! :)

**Comment by bazzadazza72** [comment]
No problem. I can build a version for you to test if you want, but I'm unsure if it'll load properly, if at all.

**Comment by tenda96** [comment]
Nono, i trust you! I think is more important the project itself than this request.
If you think that is an easy build, and you want to try to build ok, i am the first to try it!
But i don't want to waste your free time on something that will not work anyway :)

**Comment by bazzadazza72** [comment]
That's all good. We'll add it on top of Android TV, because it'll likely share the same layout. Once we have a version to test, you'll be the first to know.

**Comment by tenda96** [comment]
Perfect <3

**Comment by UkeHa** [comment]
+1 on adding android tv support!
